easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,59 @@
+# Author: Jasper Grimm (UoY)
+easyblock = 'Bundle'
+
+name = 'Wayland'
+version = '1.20.0'
```

Comment:
@sassy-crick Any reason you're sticking to 1.20.0 when 1.21.0 is available already too?

Reply:
see https://lists.freedesktop.org/archives/wayland-devel/2022-June/042268.html

Reply:
The idea was to have a bit of 'headroom' as I was not using the latest toolchain version. Thus, if somebody needs it for the latest toolchain, we don't have the same version with different toolchains. That said, following our discussions I have upgraded that. Thanks for the help.